forms/login.json

```diff
@@ -20,6 +20,18 @@
             "validation": "required|min:3",
             "class": "txt-password",
             "wrapperClass": "txt-password-wrapper"
+        },
+        {
+            "name": "gender",
+            "label": "Gender",
+            "type": "radio",
+            "option": {
```

Comment:
@yasikahivin Let's change this to `options` plural instead of `option`.

Reply:
noted @cie6868